src/algebra/group/units.lean

```diff
@@ -66,10 +67,10 @@ ext.eq_iff.symm
   mul_left_inv := λ u, ext u.inv_val }
 
 variables (a b : units α) {c : units α}
-@[simp, to_additive] lemma coe_mul : (↑(a * b) : α) = a * b := rfl
-@[simp, to_additive] lemma coe_one : ((1 : units α) : α) = 1 := rfl
-@[to_additive] lemma val_coe : (↑a : α) = a.val := rfl
-@[to_additive] lemma coe_inv : ((a⁻¹ : units α) : α) = a.inv := rfl
+@[simp, to_additive, norm_cast] lemma coe_mul : (↑(a * b) : α) = a * b := rfl
```

Comment:
Does `to_additive` copy attributes? If so, does `norm_cast` have to be applied first? If not, should it be added to the additive version too?

Reply:
I've updated the PR so the additive versions have `norm_cast` attributes. I couldn't get `to_additive` to transfer the attribute, I tried naively adding `` `norm_cast `` to the list of attributes in the `to_additive` file, but this didn't work. Adding the `norm_cast` attribute adds ` bunch of different attributes which all have to be transferred.

Reply:
That's why `norm_cast` and `simps` are not in this list.

Reply:
I guess this is ready to merge then.